src/Portal.tsx

```diff
@@ -32,6 +34,13 @@ const Portal = forwardRef<PortalRef, PortalProps>((props, ref) => {
   });
 
   useEffect(() => {
+    // Restore container to original place
```

Comment:
Patch ref comment here in case other people want to know why:

https://reactjs.org/blog/2022/03/29/react-v18.html#new-strict-mode-behaviors